src/clp_ffi_py/.clang-tidy

```
@@ -0,0 +1,6 @@
+InheritParentConfig: true
+
+CheckOptions:
+  # Variable naming rules
+  # NOTE: we set this to allow PyObject global variables to start with `Py_`
+  readability-identifier-naming.GlobalVariableIgnoredRegexp: "Py_.*"

```

```suggestion
  readability-identifier-naming.GlobalVariableIgnoredRegexp: "Py(_[a-z0-9]+)+"
```

The current regex is not restrictive enough.